core/src/main/java/org/infinispan/marshall/persistence/impl/PersistenceContextInitializer.java

```diff
@@ -40,7 +43,12 @@
             SimpleClusteredVersion.class,
             MarshallableUserObject.class,
             PrivateMetadata.class,
-            InDoubtTxInfo.class
+            InDoubtTxInfo.class,
+            MarshallableUserObject.class,
+            IracEntryVersion.class,
+            IracEntryVersion.MapEntry.class,
+            TopologyIracVersion.class,
+            IracMetadata.class
```

Comment:
I assume `InDoubtTxInfo` will be removed when this is rebased on https://github.com/infinispan/infinispan/pull/8290

Reply:
yes